lib/hammer_cli/ssloptions.rb

```diff
@@ -0,0 +1,27 @@
+module HammerCLI
+  def self.get_ssl_options
```

Comment:
This method is quite complex. Could you add a unittest? Either a standalone one or you can test it as part of the `ApiConnection` class in `test/unit/apipie/api_connection_test.rb `.